requirements.txt

```diff
@@ -15,7 +15,7 @@ future==0.16.0
 hca==4.4.1
 dcplib==1.5.1  # Recent upgrade to 1.6.0 broke the Travis build
 git+git://github.com/hannes-ucsc/jmespath.py@hannes-fix-cache-race#egg=jmespath
-git+git://github.com/HumanCellAtlas/metadata-api@release/1.0b20#egg=hca-metadata-api
```

Comment:
This can be changed to refer to the [release](https://github.com/HumanCellAtlas/metadata-api/releases/tag/release%2F1.0b21) I just cut.